### ISSUE TITLE
404 page

### ISSUE THREAD

**Issue by stursberg** [hpi-swt2/sport-portal#365]
**This depends on #378 for card component**

Layout for 404 page.

- Footer is not for us
- Cards should not be collapsible in mobile version
- Don't use the text shown in cards but the text from #378

Tasks:
- [x] change routes
- [x] convert current 404 to `.erb` and render header and footer partial as well as random text as content
- [x] fill in content from screenshots
- [x] adapt to mobile (see #182)

Now it's look like:

![Screen Shot 2018-01-16 at 19.19.19.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77226455f56b7191a379e60c2a58415e47be5d4a1f06a3a3dc78e2a436628f0d0beed23f0b26933a43a9b8841fbf02b325df1e87e7ba9850d2aa6c7bb3d4d216744b470a83ed5c32aa984ec95ef749e74a5da1123562f43b.png)

It should look like this:

![sms_02.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46fb99891cd2d1ebc3abaf51ab57f5973d047d317c05f3782387e72aba1579545f594eac42515a10adfbd82db4aa2e32db025fb28c341521842f1bbeb98515be04bb268a4d82b7b58e4bc1e12f68.png)

![sms_06.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46fb99891cd2d5ebc3abaf51ab57f5973d047d317c05f3782387e72aba1579545f594eac42515a10adfbd82db4aa2e32db025fb28c341521842f1bbeb98a15be04bb268a4d82b2bd884ec1ec2f68.png)

Header und Footer here are out of scope!

**Comment by jkbe** [comment]
**Estimation: M (Actual: M)**
